### PR TITLE
Added tool to copy attachments and statements from a contest

### PR DESCRIPTION
**Pull request by franv314**

### CODE REVIEW

File: src/tools/copy_competition_files.rs [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+#[derive(Parser, Debug, Clone)]
+pub struct CopyCompetitionFilesOpt {
+    /// Directory of the context.
+    ///
+    /// When not specified, . is assumed.
+    #[clap(short = 'c', long = "contest-dir", default_value = ".")]
+    pub contest_dir: PathBuf,
+
+    /// Look at most for this number of parents for searching the task
```
[the remark this answers is not in the document]
Comment (veluca93) [reply src/tools/copy_competition_files.rs:21]: Not that I know of.

[the remark this answers is not in the document]
Comment (franv314) [reply src/tools/copy_competition_files.rs]: Fixed

File: task-maker-format/src/ioi/task_info.rs [hunk 2]
```diff
@@ -117,7 +122,7 @@ impl IOITaskInfo {
                 .join("att")
                 .read_dir()
                 .map(|dir| {
-                    dir.filter(|entry| entry.as_ref().unwrap().file_type().unwrap().is_file())
+                    dir.filter(|entry| is_file_or_symlink(entry.as_ref().unwrap().file_type().unwrap()))
```
Comment (veluca93) [inline task-maker-format/src/ioi/task_info.rs:125]: I assume being a symlink is not enough - rather, either a file or a symlink to a file, right?

File: src/tools/copy_competition_files.rs [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+        dry_run: opt.execution.dry_run,
+    };
+
+    // create folder for competition files
+    let competition_files_dir = opt.contest_dir.join("competition-files");
+    if !competition_files_dir.exists() {
+        create_dir(&competition_files_dir)?;
+    }
+
+    for (mut task, booklets) in get_tasks_from_contest_dir(&opt.contest_dir, &opt, &eval_config)? {
```
[the remark this answers is not in the document]
Comment (franv314) [reply src/tools/copy_competition_files.rs]: solutions are explicitly not included by `eval_config`.

File: task-maker-format/src/ioi/task_info.rs (hunk 2 above)
Comment (franv314) [reply task-maker-format/src/ioi/task_info.rs]: Fixed, turns out `PathBuf::is_file` already traverses symlinks.

**Pull request merged**